Functions.Templates/Templates/DurableFunctionsOrchestration-CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,62 @@
+{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function",
+    "Durable Functions Orchestration"
+  ],
+  "name": "DurableFunctionsOrchestration",
+  "identity": "Azure.Function.CSharp.Isolated.DurableFunctionsOrchestration.1.x",
+  "groupIdentity": "Azure.Function.DurableFunctionsOrchestration",
+  "shortName": "durable",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "sourceName": "DurableFunctionsOrchestrationCSharp",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    }
+  },
+  "primaryOutputs": [
+    {
+      "path": "DurableFunctionsOrchestrationCSharp.cs"
+    }
+  ],
+  "defaultName": "DurableFunctionsOrchestrationCSharp",
+  "postActions": [
+    {
+      "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.DurableTask NuGet package",
+      "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+      "ContinueOnError": "true",
+      "ManualInstructions": [],
+      "args": {
+        "referenceType": "package",
+        "reference": "Microsoft.Azure.Functions.Worker.Extensions.DurableTask", "version": "1.0.0-rc.1",
+        "projectFileExtensions": ".csproj"
+      }
+    },
+    {
+      "description": "Restore NuGet packages required by this project.",
```

Comment:
This description ends in a period, but the others don't. Which is correct?

Reply:
Added a period to the other descriptions.